lewis/scripts/control.py

```diff
@@ -45,7 +45,10 @@ def show_api(remote, object_name):
     maxlen = len(max(properties, key=len))
     for prop in sorted(properties):
         try:
-            current_value = getattr(obj, prop)
+            raw_value = str(getattr(obj, prop))
+            value_lines = raw_value.split('\n')
+
+            current_value = value_lines[0] + (' [...]' if len(value_lines) > 1 else '')
```

Comment:
Good point, although 20 seems a bit short...maybe 40? So in total we're still below the magical command line 80 :)